.github/workflows/build_latex.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         file ./v2/english/augur-whitepaper-v2.pdf | grep -q ' PDF '
     - name: Bump version and push tag
       id: bump_version
-      uses: anothrNick/github-tag-action@1.26.0
+      uses: anothrNick/github-tag-action@1.39.0
```

Comment:
Can we use a git commit sha instead of version number?  That way the author cannot MITM us (not that it is a big worry for this project, but just a good practice).  We can leave a comment in here what the vercsion number is.